ProjectLighthouse/Database.cs

```diff
@@ -189,14 +192,16 @@ public async Task<bool> PostComment(int userId, int targetId, CommentType type,
                 .Select(u => u.UserId)
                 .FirstOrDefaultAsync();
             if (targetUserId == 0) return false;
+            if (await this.IsBlocked(userId, targetUserId)) return false;
         }
         else
         {
-            int targetSlotId = await this.Slots.Where(s => s.SlotId == targetId)
+            Slot? targetSlot = await this.Slots.Where(s => s.SlotId == targetId)
                 .Where(s => s.CommentsEnabled && !s.Hidden)
-                .Select(s => s.SlotId)
                 .FirstOrDefaultAsync();
-            if (targetSlotId == 0) return false;
+            if (targetSlot == null) return false;
+            
+            if (await this.IsBlocked(userId, targetSlot.CreatorId)) return false;
```

Comment:
```suggestion
            if (await this.IsUserBlockedBy(userId, targetSlot.CreatorId)) return false;
```
To go with the function rename